### ISSUE TITLE
Set up Transifex for Maptime

### ISSUE THREAD

**Issue by almccon** [maptime/maptime-admin#128]
@qyu is translating Maptime materials to Chinese. Is [Transifex](http://transifex.com) the best approach? Can we set this up for maptime.io, and also have separate transifex projects for tutorials, too?

**Comment by qyu** [comment]
Transifex is really user friendly. I know fieldpaper and HOT is on it. And I am working on these project also.

**Comment by jinlong25** [comment]
@qyu, do you have a link to the project on Transifex? I'm not quite familiar with it but would love to try it out!

**Comment by qyu** [comment]
We haven't set up Transifex for the Maptime yet, but you could try the LearnOSM project. It's under the HOT project.

https://www.transifex.com/hotosm/

Log in · Transifex
Log in to your Transifex account. Or sign up to start your own project or help translate an existing one.
Read more...<https://www.transifex.com/hotosm/>




________________________________

**Comment by jinlong25** [comment]
thanks @qyu, just sent the request to join the group.

**Comment by geobrando** [comment]
See https://github.com/maptime/maptime.github.io/issues/56

Set up a maptime transifex organization.  I can add HQ folks as admins, and others as maintainers or translators as requested.  https://www.transifex.com/maptime/

Made a project for the website but then deleted as there's a choice to be made between uploading source files as resources for translation and what they call their new ["Live" option](http://docs.transifex.com/tutorials/live/) which is supposed to be better for websites.

Not really sure how all of this works, but here are some resources:
* https://developmentseed.org/blog/multilingual-jekyll-sites/
* https://www.sylvaindurand.org/making-jekyll-multilingual/
* https://www.transifex.com/blog/2015/how-to-use-git-to-track-changes-in-translation-files/
* http://nicoespeon.com/en/2014/04/multilingual-blog-with-jekyll-1-5/
* The [LearnOSM repo](https://github.com/hotosm/learnosm) and their [Transifex account](https://www.transifex.com/hotosm/learnosm-1/)

**Comment by almccon** [comment]
Hi @geobrando, I tried checking out the maptime organization on transifex, and that link gives a 404.

**Comment by geobrando** [comment]
@almccon Hmmm.. Their server must have been down.  I know they've been working on a redesign. Works fine now.  I created a project for the site https://www.transifex.com/maptime/maptimeio/

**Comment by almccon** [comment]
You're right. Works now. Cool! Yeah, please add me as an admin. I'll do what I can to help facilitate maintainers and translators.
